server/blob/blob-export-file/src/test/java/org/apache/james/blob/export/file/FileSystemExtension.java

```diff
@@ -43,7 +43,9 @@ public void beforeAll(ExtensionContext context) {
 
     @Override
     public void afterAll(ExtensionContext context) throws Exception {
-        FileUtils.forceDelete(fileSystem.getBasedir());
+        if (fileSystem.getBasedir().exists()) {
```

Comment:
could you add a test for this case?

Reply:
I don't understand the reason to write test for a block of code in the test extensions. `FileUtils.forceDelete()` will throw if the File does not exist and many tests don't use `fileSystem.baseDir()` - then they don't create that baseDir. That's why I wrote a check to avoid tests failing from clean up step.

Reply:
having tests on an extension looks like a good idea: it means we can use it without worrying about corner cases.